opm/simulators/timestepping/SimulatorTimer.cpp

```diff
@@ -76,7 +76,10 @@ namespace Opm
         }
 
         setCurrentStepNum(report_step);
-        start_date_ = boost::posix_time::from_time_t(schedule.getStartTime()).date();
+        if ((boost::posix_time::from_time_t(schedule.getStartTime()) <= boost::posix_time::ptime(boost::posix_time::max_date_time)) &&
+            (boost::posix_time::ptime(boost::posix_time::min_date_time) <= boost::posix_time::from_time_t(schedule.getStartTime()))) {
```

Comment:
I appreciate the sentiment, but this doesn't do anything useful.  If the start time is outside the representable range of a `boost::posix_time`, then the `start_date_` will be "not_a_date" and that will cause all kinds of problems.  If we really want this check, then the simulation should stop if the start time is outside the representable range.